libraries/chain/kv_context_rocksdb.cpp

```diff
@@ -0,0 +1,315 @@
+#include <eosio/chain/combined_database.hpp>
+#include <eosio/chain/controller.hpp>
+#include <eosio/chain/exceptions.hpp>
+#include <eosio/chain/kv_chainbase_objects.hpp>
+#include <eosio/chain/kv_context.hpp>
+
+namespace eosio { namespace chain {
+   struct kv_iterator_rocksdb : kv_iterator {
+      uint32_t&                num_iterators;
+      b1::chain_kv::view&          view;
+      uint64_t                 contract;
+      b1::chain_kv::view::iterator kv_it;
+
+      kv_iterator_rocksdb(uint32_t& num_iterators, b1::chain_kv::view& view, uint64_t contract, const char* prefix,
+                          uint32_t size)
+          : num_iterators(num_iterators), view{ view }, contract{ contract }, //
```

Comment:
Can we make it consistent between either uniform initialization or the pre modern style?